### ISSUE TITLE
Cloudflare and LocalXpose not working

### ISSUE THREAD

**Issue by Herqules** [htr-tech/zphisher#762]
### Which environment you are using ?

None

### Describe the bug

Using MacOs. That may explain away all my issues but the masking function for clouldflare and loxalxpose do not work.

**Comment by mrfreeman1111** [comment]
Zphisher

**Comment by Herqules** [comment]
Yes, that's what im trying to execute. Ngrok works regularly on my mac as does cloudflare but not w zphisher. the only thing that currently works is using local host and then url shorterning. Maybe, if there was a URL masking feature but that is probably for a separate program.
